deploy-as-code/helm/charts/health-services/plan-service/values.yaml

```diff
@@ -0,0 +1,96 @@
+# Common Labels
+labels:
+  app: "plan-service"
+  group: "health"
+
+namespace: health
+
+# Ingress Configs
+ingress:
+  namespace: egov
+  enabled: true
+  zuul: true
+  context: "plan-service"
+
+# Init Containers Configs
+initContainers:
+  dbMigration:
+    enabled: true
+    schemaTable: "plan_schema"
+    image:
+      repository: "plan-service-db"
+
+# Container Configs
+image:
+  repository: "plan-service"
+replicas: "1"
+healthChecks:
+  enabled: true
+  livenessProbePath: "/plan-service/health"
+  readinessProbePath: "/plan-service/health"
+appType: "java-spring"
+tracing-enabled: true
+heap: "-Xmx192m -Xms192m"
+java-args: ""
+persist-yml-path: "file:///work-dir/health-campaign-config/egov-persister/plan-service-persister.yml"
+egov-mdms-service-host-key: "egov-mdms-service"
+
+# Additional Container Envs
+env: |
+  - name: SPRING_KAFKA_CONSUMER_GROUP_ID
+    value: plan-service
+  - name: SPRING_KAFKA_PRODUCER_KEY_SERIALIZER
+    value: org.apache.kafka.common.serialization.StringSerializer
+  {{- if index .Values "global" "tracer-errors-provideexceptionindetails" }}
+  - name: TRACER_ERRORS_PROVIDEEXCEPTIONINDETAILS
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: tracer-errors-provideexceptionindetails
+  {{- end }}
+  - name: STATE_LEVEL_TENANT_ID
+    valueFrom:
+      configMapKeyRef:
+        name: egov-config
+        key: egov-state-level-tenant-id 
+  - name: EGOV_IDGEN_HOST
```

Comment:
why is ID gen related properties required?

Reply:
have removed them.